community/kernel/src/main/java/org/neo4j/kernel/impl/locking/Locks.java

```diff
@@ -78,14 +78,14 @@ void visit( ResourceType resourceType, long resourceId, String description, long
          * Can be grabbed when there are no locks or only share locks on a resource. If the lock cannot be acquired,
```

Comment:
Now `Locks` interface looks weird: we allow acquisition of multiple resources but do not allow released them together.